glossary/es/eu-psi-directive/index.md

```diff
@@ -0,0 +1,9 @@
+---
+section: terms
+lang: es
+title: Directiva de la Unión Europea sobre Información del Sector Público
+---
+
+El término Directiva de la Unión Europea sobre Información del Sector Público reconoce la importancia de los registros más atractivos para posibles reutilizadores de información del sector público. Requiere que los estados miembro provean listas, portales o algo similar. Dice:
```

Comment:
I didn't do the translation, I'm working with the original previous content, I'm focusing at the moment on giving the correct structure to the glossary section on spanish, when it is ok, my plan is to finish and correct the translation …